### DISCUSSION TITLE
Show permalinks on hover

### DISCUSSION THREAD

**Discussion by jermanuts** [withastro/starlight#1525] [Feature Requests]
### What version of `starlight` are you using?

0.19.0

### What is your idea?

When hovering over Headings tags it would show a # or  any indicator.

![image](https://github.com/withastro/starlight/assets/109705802/8ce908da-fc75-45e8-8dea-8414c2fb434c)

![image](https://github.com/withastro/starlight/assets/109705802/592ed64d-3066-4a60-9e0f-514667fdca23)

### Why is this feature necessary?

Very convenient than using the Table of contents on the right. 

### Do you have examples of this feature in other projects?

https://docusaurus.io/docs

https://squidfunk.github.io/mkdocs-material/getting-started/

### Participation

- [ ] I am willing to submit a pull request for this feature.

**Comment by at-the-vr** [comment]
Hey jermanuts 👋 Here's a quick guide to add those heading links to your site: https://hideoo.dev/notes/starlight-heading-links

**Comment by delucis** [comment]
Thanks for the suggestion @jermanuts — and thanks for the link @at-the-vr!

We’re actually already tracking this idea in #1239, so will close this as a duplicate of that. Please do upvote that feature request as well — it helps us evaluate what to work on next.
